app/models/dashboard.rb

```diff
@@ -40,7 +41,20 @@ def closed_defence_requests
   attr_reader :user, :defence_requests_scoped_by_policy
 
   def ordered_defence_requests
-    defence_requests_scoped_by_policy.ordered_by_created_at
+    defence_requests_scoped_by_policy.ordered_by_created_at.map { |d| new_dash_dr[d] }
+  end
+
+  def new_dash_dr
+    @_new_dash_dr ||= DefenceRequest.method(:new).curry(2)[@client]
```

Comment:
I'm not entirely sure on this approach. Currying is fine, albeit a "strange" idiom in Ruby. However, here it is used in support of the nested decorator/presenter type object that doesn't seem to be pulling much weight?
I'm OK with going with this approach until we have more info to make a decision on the decorator, but I'd still like to hear more comments from the rest of the team.